File: C++/Arrays and Hashing/TwoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> seen; 
        vector<int> answer; 

        for(int i = 0; i < nums.size(); i++){
            int lookup = target - nums[i]; 
            if(seen.count(lookup)){
                answer.push_back(i); 
                answer.push_back(seen[lookup]);
                return answer; 
            }
            seen[nums[i]] = i; 
        }
        return {}; 
    }
};

File: Python/Arrays and Hashing/TwoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        seen = {}

        for i, num in enumerate(nums):
            lookup = target - num 
            if lookup in seen:
                return[i, seen[lookup]]
            seen[num] = i  

        return []       